test/functional/questions_search_controller_test.rb

```diff
@@ -8,15 +8,15 @@ def setup
     activate_authlogic
   end
 
-  test "should get questions_search and render template if question match found" do
-    get :index, id: 'How to'
-    assert_response :success
-    assert_not_nil :title
-    assert_not_nil :tagnames
-    assert_not_nil :users
-    assert_not_nil :notes
-    assert_template :index
-  end
+  #test "should get questions_search and render template if question match found" do
```

Comment:
@david-days @ujithaperera The current question search in the questions uses this controller. You can remove this if you have implemented this part.